packages/scene-system/scene.system.ts

```diff
@@ -240,10 +242,10 @@ export default class GamekitScene extends Script {
           if (componentNameRE.test(componentName)) {
             that.events.push({
               type: 'AttachEntityComponent',
+              id: id,
```

Comment:
You can just use `id`, is called object property short-hand, info: https://alligator.io/js/object-property-shorthand-es6
```suggestion
              id,
```